app-modules/etno/src/Filament/Schemas/Components/ProvenanceAndResearchContextSection.php

```
@@ -0,0 +1,31 @@
+    {
+        parent::setUp();
+
+        $this->heading('Provenance and Research Context')
+            ->schema([
+                InstitutionSelect::make('institution_id')
+                    ->columnSpanFull(),
+                ProjectSelect::make('project_id')
+                    ->columnSpanFull(),
+                CollectionMethodSelect::make('collection_method'),
+                AccrualMethodSelect::make('accrual_method'),
+                TimePeriodSection::make('Time period'),
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'Metafori\Etno\Filament\Schemas\Components\TimePeriodSection' in method 'setUp'. (undefined)

(StaticAccess)